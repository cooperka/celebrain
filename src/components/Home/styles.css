.Home .buttons {
  margin: 1rem auto;
}
