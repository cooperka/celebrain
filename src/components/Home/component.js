// @flow

import React from 'react';

import NavButton from '../NavButton/index';

import './styles.css';

function Home() {
  return (
    <div className="Home page">
      <div className="title">Celebrain</div>
      <div className="section">
        <div className="subtitle">
          Improve your memory by matching celebrity names to faces.
        </div>
      </div>
      <div className="buttons">
        <NavButton bordered linkTo="/game">Start!</NavButton>
      </div>
    </div>
  );
}

export default Home;
