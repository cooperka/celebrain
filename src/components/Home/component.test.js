// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import Home from './component';

it('renders', () => {
  const tree = renderer.create(<Home />);
  expect(tree.toJSON()).toMatchSnapshot();
});
