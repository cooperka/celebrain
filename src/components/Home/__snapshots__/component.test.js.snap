// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="Home page"
>
  <div
    className="title"
  >
    Celebrain
  </div>
  <div
    className="section"
  >
    <div
      className="subtitle"
    >
      Improve your memory by matching celebrity names to faces.
    </div>
  </div>
  <div
    className="buttons"
  >
    <NavLink
      className="MuiButtonBase-root-16 MuiButton-root-1 NavButton bordered"
      exact={true}
      href={undefined}
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={0}
      to="/game"
    >
      <span
        className="MuiButton-label-2"
      >
        <div
          className="link-text"
        >
          Start!
        </div>
      </span>
      <span
        className="MuiTouchRipple-root-18"
      />
    </NavLink>
  </div>
</div>
`;
